OrbitQt/orbitglwidget.cpp

```diff
@@ -41,16 +41,15 @@ bool OrbitGLWidget::eventFilter(QObject* /*object*/, QEvent* event) {
 
 void OrbitGLWidget::Initialize(GlPanel::Type a_Type,
                                OrbitMainWindow* a_MainWindow) {
-  UNUSED(a_Type);
-  UNUSED(a_MainWindow);
-
   m_OrbitPanel = GlPanel::Create(a_Type);
```

Comment:
Should this just be unique_ptr? and modify GlPanel::Create to return unique_ptr as well?

Reply:
That was also my first change, but this caused Orbit to crash... I didn't spend a lot of time looking into it though, if you want I can have another look at this.

Reply:
It is strange.. It would be good to know why Orbit crashes in case of unique_ptr, I do not see anything which could imply a different scope.

Reply:
Ok, so the short answer is: Because I did it wrong :-) I changed the code to return a unique_ptr as you suggested, this is actually cleaner and requires a lot less code... thanks for pushing for it!